### PR TITLE
enex2notion 0.2.17 (new formula)

### PR DESCRIPTION
**Pull request by vzhd1701**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/enex2notion.rb [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+            <![CDATA[<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+              <!DOCTYPE en-note SYSTEM "http://xml.evernote.com/pub/enml2.dtd">
+              <en-note><div><br /></div></en-note>
+            ]]>
+          </content>
+        </note>
+      </en-export>
+    EOF
+
+    system bin/"enex2notion", "test.enex"
```
Comment (SMillerDev) [inline Formula/enex2notion.rb:159]: Does this output anything we can check?

Comment (vzhd1701) [reply Formula/enex2notion.rb:159]: It outputs "No token provided" warning, I'll add it to the test.

File: Formula/enex2notion.rb [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+  resource "requests" do
+    url "https://files.pythonhosted.org/packages/a5/61/a867851fd5ab77277495a8709ddda0861b28163c4613b011bc00228cc724/requests-2.28.1.tar.gz"
+    sha256 "7c5599b102feddaa661c826c56ab4fee28bfd17f5abca1ebbe3e7f19d7c97983"
+  end
+
+  resource "six" do
+    url "https://files.pythonhosted.org/packages/71/39/171f1c67cd00715f190ba0b100d606d440a28c93c7714febeca8b79af85e/six-1.16.0.tar.gz"
+    sha256 "1e61c37477a1626458e36f7b1d82aa5c9b094fa4802892072e49de9c60c4c926"
+  end
```
Comment (carlocab) [inline Formula/enex2notion.rb:91]: Can we use
```ruby
depends_on "six"
```
instead?

Comment (vzhd1701) [reply Formula/enex2notion.rb]: Fixed

**Review by carlocab — APPROVED** [review]
LGTM. Btw, you may wish to use `tap_migrations.json` to migrate users who have installed a formula from your tap onto homebrew/core.

**Pull request closed without merging**